.github/workflows/FirebaseDeploy.yml

```
@@ -0,0 +1,58 @@
+
+      - name: Authenticate Firebase
+        env:
+          FIREBASE_SERVICE_JSON: ${{ secrets.FIREBASE_SERVICE_JSON }}
+        run: |
+          echo $FIREBASE_SERVICE_JSON | base64 --decode > functions/firebase-service-account.json
+          firebase auth:login:ci --token "$(firebase login:ci --json --project ${{ secrets.FIREBASE_PROJECT_ID }})"
+
+      - name: Deploy Firestore Rules and Indexes
+        env:
+          FIREBASE_PROJECT_ID: ${{ secrets.FIREBASE_PROJECT_ID }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:69: Double quote to prevent globbing and word splitting

(shellcheck)